package.json

```diff
@@ -2,31 +2,16 @@
   "name": "yargs",
   "version": "17.7.2",
   "description": "yargs the modern, pirate-themed, successor to optimist.",
-  "main": "./index.cjs",
+  "main": "./index.mjs",
   "exports": {
     "./package.json": "./package.json",
-    ".": [
```

Comment:
Still need an export of `"."` to get access to `index.mjs`.

Reply:
👍 thanks reading this as I work through things.

Reply:
Added back ".".